### DISCUSSION TITLE
TIL params inheritance strategy to always have access to parent route info

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#313] [Announcements]
# TIL params inheritance strategy to always have access to parent route info

Configure the Angular router to always have access to the parent route data from within the child route.

https://timdeschryver.dev/blog/til-paramsinheritancestrategy-to-always-have-access-to-parent-route-info

**Comment by Frankitch** [comment]
Not the first time you save my life.
Thanks for all your contributions!

**Reply by timdeschryver** [reply]
Thanks @Frankitch !

**Reply by johnschlee** [reply]
Agreed, it was very helpful. I was starting to find myself up a creek without a paddle with router store + nested router outlets.

**Comment by matheo** [comment]
Just for the record:
```
  provideRouter(
    appRoutes,
    withRouterConfig({ paramsInheritanceStrategy: 'always' }),
  ),
```

**Comment by Loaak** [comment]
Hello, thanks for the article ! 
Just to let you know about a tiny typo : 
`Now, to access the parent id (comanyId) from the customers component, the code that I used to write, and have seen a lot, is the following one` : comanyId -> companyId

Thanks again for this article :)

**Reply by timdeschryver** [reply]
Woops, thanks for the catch!
